app/Models/SavedSearch.php

```diff
@@ -26,4 +29,15 @@ public function getOwnerName()
     {
         return $this->user ? $this->user->name : '';
     }
-}
\ No newline at end of file
+
+    public function recordtype()

```

Comment:
Change the method name to camelCase, and add comment.

Reply:
We might need to create another issue for this one because all Models are using this name so there are a lot of changes needed.